src/pdb/mod.rs

```diff
@@ -513,74 +540,55 @@ impl RowGroup {
         &self,
         writer: &mut W,
         options: &WriteOptions,
-        args: (u64, u64),
+        args: (u64, u64, u16),
     ) -> binrw::BinResult<u64> {
         let options = options.with_endian(Endian::Little);
 
-        let (page_offset, relative_row_offset) = args;
+        let (page_offset, relative_row_offset, rows_in_group) = args;
```

Comment:
you can just use `self.rows.len()` instead of passing `rows_in_group` through here.